client/src/App.js

```diff
@@ -4,21 +4,26 @@ import ProtectedRoute from './components/routes/ProtectedRoute';
 import ChessGame from './components/chess-game/ChessGame';
 import Home from './components/home/Home';
 import Login from './components/login/Login';
-import Dashboard from './components/dashboard/Dashboard';
+// import Dashboard from './components/dashboard/Dashboard';
 import Leaderboard from './components/leaderboard/Leaderboard';
+import MenuBar from './components/menu-bar/MenuBar';
 
 function App() {
   return (
-    <Router>
-      <Switch>
-        <Route path="/login" exact component={Login} />
-        <Route path="/chess-game" exact component={ChessGame} />
-        <Route path="/" exact component={Home} />
-        <ProtectedRoute path="/dashboard" exact component={Dashboard} />
-        <ProtectedRoute path="/leaderboard" exact component={Leaderboard} />
-        <Route path="*" component={() => '404 NOT FOUND'} />
-      </Switch>
-    </Router>
+    <>
+      {/* <MenuBar /> */}
+      <Router>
+        <Route component={MenuBar} />
+        <Switch>
+          <Route path="/" exact component={Login} />
+          <ProtectedRoute path="/dashboard" exact component={Home} />
```

Comment:
I think we should have `/login` for login
And `/` redirect to `/dashboard`.
This here is a bit confusing.

Reply:
I think, we should have "/" for our site and as our home page is Login we should keep it "/" as login component.
"/dashboard" is rendering Home component that is confusing, I will change component's name to Dashboard